plugins/@grouparoo/sqlite/src/lib/sqlite.ts

```diff
@@ -0,0 +1,48 @@
+import { Database } from "sqlite3";
+
+export class SQLite extends Database {
+  database: string;
+  connection: Database;
+
+  constructor({ database }) {
+    super(database);
+    this.database = database;
+    this.connection = undefined;
+  }
+
+  async asyncConnect(): Promise<string> {
+    return new Promise((resolve, reject) => {
+      this.connection = new Database(this.database, (err: Error) => {
+        if (err) {
+          return reject(
+            new Error(`Error connecting to SQLite database: ${this.database}`)
+          );
+        }
+        return resolve(
+          `Successfully connected to SQLite database: ${this.database}`
+        );
+      });
+    });
+  }
+
+  async asyncQuery(query: string): Promise<Array<any>> {
+    return new Promise((resolve, reject) => {
+      this.connection.all(query, (err: Error, res: any[]) => {
+        if (err) {
+          return reject(new Error(`Error running query: ${err}`));
```

Comment:
I think you'll end up with `Error running query: Error: foo` I think.   Perhaps `new Error(`Error running query: ${err.message}`)`

Reply:
I ended up printing the error first and then the query second. Thought that could come in handy, as it's likely to be a syntax goof or something  I did.